homeassistant/components/config/automation.py

```diff
@@ -92,10 +107,10 @@ def _write_value(self, hass, data, config_key, new_value):
 
 
 @websocket_api.websocket_command(
```

Comment:
Decorate all commands with `@websocket_api.require_admin`

Reply:
Fixed in bc24777 for the breakpoints related commands.
Should we require admin access also to view and list traces?

Reply:
Yes